docs/_docs/trend_changepoints.md

```diff
@@ -22,7 +22,7 @@ Even though we have a lot of places where the rate can possibly change, because
 The number of potential changepoints can be set using the argument `n_changepoints`, but this is better tuned by adjusting the regularization.
 
 ### Adjusting trend flexibility
-If the trend changes are being overfit (too much flexibility) or underfit (not enough flexiblity), you can adjust the strength of the sparse prior using the input argument `changepoint_prior_scale`. By default, this parameter is set to 0.05. Increasing it will make the trend *more* flexibile:
```

Comment:
@pborreli @seanjtaylor 

I think I've found where the error comes from.

It's actually the `jupyter nbconvert` that seems to create the `xxx.md` file and the `xxx_files` directories in the same directory as that of the original notebook. Basically, it creates them in `../notebooks/`.

Change the paths in the makefile as follows and it should work:
```
for f in ../notebooks/*.ipynb; \
        do \
            NAME=$$(basename $$f .ipynb); \
            jupyter nbconvert --to markdown ../notebooks/$$NAME.ipynb --template=nbconvert_template.tpl; \
            mv -f ../notebooks/"$$NAME".md _docs/; \
            rm -rf static/"$$NAME"_files; \
            mv ../notebooks/"$$NAME"_files static/; \
        done
```